docs/inso-cli/cli-command-reference/inso-generate-config.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: article-detail
+title:  inso generate config
+category: "CLI Command Reference"
+category-url: inso-cli
+---
+
+Similar to the Kong Kubernetes and Declarative config plugins for Insomnia, this command can generate configuration from an API specification, using openapi-2-kong.
+
+### Command
```

Comment:
Should these all be `h3` or is that just a result of copy-pasting from the single file. All good either way, double checking